doc/reference/reference_lua/http.rst

```diff
@@ -1,353 +1,1141 @@
-            will wait while the connection is idle before sending keepalive
-            probes. See also
-            `CURLOPT_TCP_KEEPIDLE <https://curl.haxx.se/libcurl/c/CURLOPT_TCP_KEEPIDLE.html>`_
-            and the note below about ``keepalive_interval``.
-          * ``keepalive_interval`` -- the interval, in seconds, that the operating
-            system will wait between sending keepalive probes. See also
-            `CURLOPT_TCP_KEEPINTVL <https://curl.haxx.se/libcurl/c/CURLOPT_TCP_KEEPINTVL.html>`_.
-            If both ``keepalive_idle`` and ``keepalive_interval`` are set, then
-            Tarantool will also set HTTP keepalive headers: ``Connection:Keep-Alive``
-            and ``Keep-Alive:timeout=<keepalive_idle>``.
-            Otherwise, Tarantool will send ``Connection:close``.
-          * ``low_speed_limit`` -- set the "low speed limit" -- the average
-            transfer speed in bytes per second that the transfer should be below
-            during "low speed time" seconds for the library to consider it to be
-            too slow and abort. See also
-            `CURLOPT_LOW_SPEED_LIMIT <https://curl.haxx.se/libcurl/c/CURLOPT_LOW_SPEED_LIMIT.html>`_.
-          * ``low_speed_time`` -- set the "low speed time" -- the time that the
-            transfer speed should be below the "low speed limit" for the library
-            to consider it too slow and abort. See also
-            `CURLOPT_LOW_SPEED_TIME <https://curl.haxx.se/libcurl/c/CURLOPT_LOW_SPEED_TIME.html>`_.
-          * ``max_header_name_len`` -- the maximal length of a header name. If a header
-            name is bigger than this value, it is truncated to this length.
-            The default value is ``32``.
-          * ``follow_location`` -- when the option is set to ``true`` (default)
-            and the response has a 3xx code, the HTTP client will automatically issue
-            another request to a location that a server sends in the ``Location``
-            header. If the new response is 3xx again, the HTTP client will
-            issue still another request and so on in a loop until a non-3xx response
-            will be received. This last response will be returned as a result.
-            Setting this option to ``false`` allows to disable this behavior.
-            In this case, the HTTP client will return a 3xx response itself.
-          * ``no_proxy`` -- a comma-separated list of hosts that do not require proxies, or '*', or ''.
-            Set :samp:`no_proxy = {host} [, {host} ...]` to specify
-            hosts that can be reached without requiring a proxy, even if ``proxy`` has
-            been set to a non-blank value and/or if a proxy-related environment variable
-            has been set.
-            Set ``no__proxy = '*'`` to specify that all hosts can be reached
+        Make an HTTP request and receive a response.
+
+        :param string method: a request HTTP method, for example, ``GET``, ``POST``, or ``PUT``
```

Comment:
I think we need all possible values in the reference, not just examples.

Reply:
agree